<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog_pr</class>
 <widget class="QDialog" name="AddDialog_pr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar projeto</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Nome da Projeto:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nome_prlineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Descricao:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="descricao_prlineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Prazo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCalendarWidget" name="prazo_calendarWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="semprazo_checkBox">
       <property name="text">
        <string>Sem Estimativa</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="guardar_pushButton">
         <property name="text">
          <string>Guardar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelar_pushButton">
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
